realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -70,7 +70,7 @@
 
     private final Table table;
     private final BaseRealm realm;
-    private final TableQuery query;
+    public final TableQuery query; // FIXME Make private somehow
```

Comment:
This one might be a bit tricky to resolve, we might need to create an accessor in the right package and then inject it into the `ObjectServerFacade`, to be later consumed by the `Subscription` factories.

Reply:
I'm not sure there is a great way to achieve this. For now, I made this private again and create a dedicated getter with explicit javadoc that this is for internal use only.